### PR TITLE
Fix sampler and copy/paste text layer with multiple text styles

### PR DESCRIPTION
**Pull request by shaoshing**
Addresses issue #3123 

If a text has multiple text styles (multiple text ranges with different text style), we will use the first character style (and the stroke color) in these tools:

* sampler
* copy/paste appearance
* create text style in libraries panel

### CODE REVIEW

File: src/js/stores/font.js [hunk 1]
```diff
@@ -237,7 +234,7 @@ define(function (require, exports, module) {
                 };
             }
 
-            if (textStyle.leading >= 0) {
```
Comment (shaoshing) [inline src/js/stores/font.js]: This will filter out `null` value in `textStyle.leading`, which is also `null >= 0`

**Comment by iwehrman** [comment]
This looks OK to me so far.

**Comment by iwehrman** [comment]
Works OK for me so far.

**Comment by iwehrman** [comment]
Well, I'm satisfied. Someone else should take a look and test independently though.

**Comment by pineapplespatula** [comment]
looks good. i'll test a little more and then merge it

**Comment by shaoshing** [comment]
Thanks everyone

**Pull request merged**